modulemd/v2/modulemd-defaults-v1.c

```diff
@@ -46,6 +47,100 @@ G_DEFINE_TYPE (ModulemdDefaultsV1,
                MODULEMD_TYPE_DEFAULTS)
 
 
+GEqualFunc profile_defaults_equals (g_autoptr (GPtrArray) profile_defaults_1,
```

Comment:
You don't need this function at all. You're calling this equivalence within `modulemd_defaults_equals()`, which means you've already processed each of the keys. This function then processes all of the keys of the set of profile defaults and then calls a `modulemd_hash_table_sets_are_equal()` on each of the values, except those values are strings, not hash tables.

Delete this function and just pass `modulemd_hash_table_sets_are_equal()` to `modulemd_defaults_equals()` below.